NSS-Day-Cohort-53/kennels-voyagers
# Ap locations

1. Modified file LocationList.js to include .then in useEffect for LocationRepository and update locations.
​
Steps to Review
1. Checkout this branch locally.
    
    git fetch --all
    git checkout ap-locations
    
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > click on locations in navbar and confirm that locations shows up.
    
4. View code file.
    > Confirm LocationList.js modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
